README.md

```diff
@@ -65,6 +65,13 @@ The software behind Wikipedia
 * license: CC BY-SA 3.0
 * [website](https://www.mediawiki.org/wiki/MediaWiki) | [repo](https://phabricator.wikimedia.org/diffusion/)
 
+### SignDict
+SignDict is a english and german dictionary for sign language (using the GSL and DGS respectively). Currently it has over 4000 signs listed.
```

Comment:
Isn't only the frontend website in both english and german? 
When looking at the dictionary itself, I cannot find the english version: https://docs.signdict.org/

Reply:
Hmmm, yeah, seems that only the website itself is in english, not the translations or documentations.
I'll remove english from this list